src/main/java/io/reflect/token/TokenUtil.java

```diff
@@ -0,0 +1,22 @@
+package io.reflect.token;
+
+import java.nio.ByteBuffer;
+import java.util.UUID;
+
+import javax.crypto.SecretKey;
+import javax.crypto.spec.SecretKeySpec;
+
+final class TokenUtil {
+    static SecretKey secretKeyFromUUID(String uuid) {
+        try {
+            UUID secretKeyUUID = UUID.fromString(uuid);
+            ByteBuffer secretKeyBytes = ByteBuffer.wrap(new byte[16]);
+            secretKeyBytes.putLong(secretKeyUUID.getMostSignificantBits());
+            secretKeyBytes.putLong(secretKeyUUID.getLeastSignificantBits());
+
+            return new SecretKeySpec(secretKeyBytes.array(), "AES");
+        } catch (IllegalArgumentException iae) {
+            throw new InvalidSecretKeyException();
+        }
+    }
+}
```

Comment:
Meh, the only utils here are related to secret keys so I'd move this to `SecretKeyUtils`